docs/architecture/dispatch.md

```diff
@@ -1,8 +1,8 @@
 # Dispatch
 
-The **dispatch** function controls how to execute [actions](actions.md) and apply the state transitions and effect running that they request to be done.
+The **dispatch** function controls Hyperapp's core dispatching process which executes [actions](actions.md), applies state transitions, runs [effects](actions.md#effects), patches the DOM, and starts/stops [subscriptions](subscriptions.md) that need it.
```

Comment:
> patches the DOM,

`dispatch` ultimately causes the DOM to be patched, but very indirectly. What motivated you to add this part?

Reply:
Hyperapp has to ultimately patch the DOM at some point during its processing, otherwise the screen would be blank. The mention of it here is just an acknowledgement of that.

Perhaps better wording is needed to show that the patching part is clearly internal functionality ?

Reply:
Or just remove the "patches the DOM," part for now.

Reply:
> Perhaps better wording is needed to show that the patching part is clearly internal functionality?

Sure, but it may be tricky to come up with the right wording, so until then, can we just go without "patches the DOM"?

Reply:
Upon reflection I'll take it out.